README.md

```diff
@@ -335,52 +335,26 @@ To compile:
 To clone the gem5-resources repository, run the following command:
 
 ```
-git clone https://gem5.googlesource.com/public/gem5-resources
+git clone https://github.com/gem5/gem5-resources
```

Comment:
I am not sure this should be part of the mmap patch, but is also something that is needed